subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1970,7 +1970,6 @@ static int light_ctrl_srv_start(const struct bt_mesh_model *model)
 		break;
 	case BT_MESH_ON_POWER_UP_RESTORE:
 		if (is_enabled(srv)) {
-			reg_start(srv);
```

Comment:
I have one questions. Are we sure that FLAG_AMBIENT_LUXLEVEL_SET is cleared? So when handle_sensor_status is run it calls the reg_start()? If not reg_start will not be called before a disable/enable is called.